docs/guide/acquiring_data/fido.rst

```diff
@@ -62,19 +62,6 @@ Using ``Instrument`` as the first example, if you print the object::
 You get a full list of known values, a description and what "Clients" support those values (if you want to use a specific data source).
 This is supported for most attributes including the client specific ones.
 
-For VSO::
```

Comment:
Are we really out of VSO specific attrs we register values for?

Reply:
Yes, though there are `Pscale`, `Extent`, `Pixel` etc, but nothing is registered for them.